icode.properties

```diff
@@ -1,12 +1,18 @@
 category=External Analyzers
-description=Import i-Code CNES results for Fortran(77 & 90) and Shell
+description=Analyze your Fortran(77 & 90) and Shell code with i-Code CNES
 homepageUrl=https://github.com/lequal/sonar-icode-cnes-plugin
 archivedVersions=1.0.2,1.1.0
-publicVersions=
+publicVersions=2.0.0
 
 defaults.mavenGroupId=fr.cnes.sonarqube.plugins
 defaults.mavenArtifactId=sonaricode
 
+2.0.0.description=i-Code CNES is now embedded in the plugin, no more need to install it
+2.0.0.date=2020-02-14
+2.0.0.sqVersions=[6.7.7,8.1]
```

Comment:
Versions below 7.9.* have been archived. Please start this compatibility range at 7.9